sdk/core/azure-core/src/main/java/com/azure/core/http/rest/RestProxy.java

```diff
@@ -180,13 +181,14 @@ public Object invoke(Object proxy, final Method method, Object[] args) {
     /**
      * Starts the tracing span for the current service call, additionally set metadata attributes on the span by passing
      * additional context information.
-     *
-     * @param method Service method being called.
+     * @param method  Service method being called.
      * @param context Context information about the current service call.
+     *
      * @return The updated context containing the span context.
      */
     private Context startTracingSpan(Method method, Context context) {
-        if (!TracerProxy.isTracingEnabled()) {
+        boolean disableTracing =  (boolean) context.getData(Tracer.DISABLE_TRACING_KEY).orElse(true);
```

Comment:
Should this be `orElse(false)` , in the cases when a user doesn't want to disable explicitly and have it enabled by default?

Reply:
Fixed.